script/test-cross-model-e2e.sh

```
@@ -0,0 +1,291 @@
+
+echo "════════════════════════════════════════════════════════════"
+echo "  Cross-Model E2E Test Suite - 5 Model Variants"
+echo "════════════════════════════════════════════════════════════"
+echo ""
+
+# Test 1: Gemini → Anthropic Claude (original bug + direct Anthropic API)
+echo "Test 1: Gemini Pro → Anthropic Claude Opus (direct API)"
+log_info "Step 1: Gemini with thinking + tool..."
+opencode run -m google/antigravity-gemini-3-pro-low \
+  "Run: echo 'Test1-Gemini'. Think about sequences." \
+  2>&1 > /tmp/e2e-t1-s1.log || true
```

Shellcheck (0.11.0)

[warning] To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

(SC2069)